app/views/DisplayPrefsView.tsx

```diff
@@ -16,12 +16,29 @@ import SafeAreaView from '../containers/SafeAreaView';
 import { ICON_SIZE } from '../containers/List/constants';
 import log, { events, logEvent } from '../utils/log';
 import { DISPLAY_MODE_CONDENSED, DISPLAY_MODE_EXPANDED } from '../constants/constantDisplayMode';
-
-const DisplayPrefsView = props => {
+import { SettingsStackParamList } from '../stacks/types';
+
+interface IParam {
```

Comment:
Are you sure about all fields is optional ? If so, we can remove all `?` and use the `Partial<IParams>`